cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,33 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-
+  it('should let user to login', () => {
+    cy.login();
+    cy.visit('/profile');
+    cy.get('#userName-value').should('contain', 'Dmytro');
+    cy.url().should('include', '/profile');
   });
 
-  it('', () => {
+  it('should add a book to the collection', () => {
+    cy.viewport(750, 600);
+    cy.login();
+    cy.visit('/profile');
+    cy.contains('.btn', 'Book Store').click();
+    cy.get('[placeholder="Type to search"]').click();
+    cy.get('[placeholder="Type to search"]').type('Speaking JavaScript');
+    cy.contains('O\'Reilly Media');
```

Comment:
Assert description of the book:   name: 'Speaking JavaScript', author: 'Axel Rauschmayer', publisher: "O'Reilly Media" (it would be nice to encapsulate it in an object for not using hardcoded data in tests)

Reply:
Thank! Fixed